lib/restbase.js

```diff
@@ -325,13 +325,20 @@ RESTBase.prototype._request = function(req) {
         });
 
         if (accessRestricted) {
-            return P.all([
-                reqHandlerPromise,
-                childRESTBase._authService.checkPermissions(childRESTBase, childReq)
-            ])
-            .then(function(res) {
-                return res[0];
-            });
+            if (req.method === 'get' || req.method === 'head') {
```

Comment:
@d00rman right, I'll move the security check to POST endpoint, but it depends on a little bug-fix in router: https://github.com/wikimedia/swagger-router/pull/21